java/test/jmri/StringIOTest.java

```diff
@@ -0,0 +1,72 @@
+package jmri;
+
+import jmri.implementation.AbstractNamedBean;
+import org.junit.*;
+
+/**
+ * Tests for the Light class
+ *
+ * @author	Daniel Bergqvist Copyright (C) 2018
+ */
+public class StringIOTest {
```

Comment:
I don't see this in PackageTest, perhaps it was overlooked?

Reply:
It's at line 28 in jmri.PackageTest.